generator/configurations.libsonnet

```diff
@@ -1,21 +1,82 @@
-local namespaced = import './namespaced.libsonnet';
 local crossplane = import 'github.com/jsonnet-libs/crossplane-libsonnet/crossplane/1.17/main.libsonnet';
 
 local configuration(key, version) =
   local conf = crossplane.pkg.v1.configuration;
   conf.new(key)
   + conf.spec.withPackage('ghcr.io/grafana/crossplane/' + key + ':' + version);
 
-local groups =
+local xrds =
+  std.map(
+    function(o) o.definition,
+    (import './namespaced.libsonnet'),
+  );
+
+local crds =
+  std.filter(
+    function(crd) crd.spec.group != 'grafana.crossplane.io',
+    std.parseYaml(importstr './crds.yaml'),
+  );
+
+local gvkXRDs =
+  std.flatMap(
+    function(definition) [
+      {
+        group: definition.spec.group,
+        version: v.name,
+        kind: definition.spec.claimNames.kind,
+        plural: definition.spec.claimNames.plural,
```

Comment:
The `plural` key is intended for consumption by the ksm library, see https://github.com/crdsonnet/kube-state-metrics-libsonnet/pull/2